StringCalculatorKata/NegativeNumbersNotAllowedException.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace StringCalculatorKata;
+public class NegativeNumbersNotAllowedException : Exception
+{
+    public NegativeNumbersNotAllowedException(IEnumerable<int> negativeNumbers)
+        : base($"Negatives not allowed: {string.Join(", ", negativeNumbers)}")
```

Comment:
plz place this error message in a separate file, allowing for consistent reuse of the same message in assertions within tests